### PR TITLE
Rename to main

### PR DESCRIPTION
**Pull request by jarrodmillman**
- [ ] see #39577
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/io/test_html.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ def test_banklist_url(self):
     @tm.network
     def test_spam_url(self):
         url = (
-            "https://raw.githubusercontent.com/pandas-dev/pandas/master/"
+            "https://raw.github.com/pandas-dev/pandas/main/"
```
Comment (mroeschke) [inline pandas/tests/io/test_html.py:174]: Nit: Could you keep `githubusercontent` here? It redirects to that domain from raw.github.com

Comment (jarrodmillman) [reply pandas/tests/io/test_html.py:174]: When I try going to https://raw.githubusercontent.com/pandas-dev/pandas/main/, I get `400: Invalid request`.  Is that what you want?

Comment (mroeschke) [reply pandas/tests/io/test_html.py:174]: The string is continuing on the next line here: https://raw.githubusercontent.com/pandas-dev/pandas/main/pandas/tests/io/data/html/spam.html

**Review by mroeschke — APPROVED** [review]

**Comment by jreback** [comment]
@meeseeksdev backport 1.4.x

**Comment by jreback** [comment]
thanks @jarrodmillman

**Pull request merged**